app/javascript/packs/application.js

```diff
@@ -4,53 +4,61 @@
 // that code so it'll be compiled.
 
 import Rails from "@rails/ujs"
-import Turbolinks from "turbolinks"
```

Comment:
Since we've gotten rid of turbolinks here, we should probably get rid of the gem as well.